tests/integration/conftest.py

```diff
@@ -56,6 +56,7 @@ def teardown():
 
         sleep(10)  # Makes sure that ES is up.
         _es = app.extensions['invenio-search']
+        list(_es.delete(ignore=[400]))
```

Comment:
The problem with doing this is that, if you `git bisect`, you get a broken build on this commit. Ideally we should be passing the suite at every commit!